src/com/facebook/buck/android/AndroidLibraryDescription.java

```diff
@@ -175,5 +176,6 @@ public boolean hasFlavors(ImmutableSet<Flavor> flavors) {
   public static class Arg extends JavaLibraryDescription.Arg {
     public Optional<SourcePath> manifest;
     public Optional<String> resourceUnionPackage;
+    public Optional<String> finalRName;
```

Comment:
Can you add docs for this so others can also know how to use ButterKnife with Buck?